app/src/main/java/net/squanchy/service/firebase/FirebaseDbService.java

```diff
@@ -37,8 +38,10 @@
     private static final String TRACKS_NODE = "data/tracks";
     private static final String VENUE_INFO_NODE = "data/venue";
     private static final String TRACKS_BY_ID_NODE = "data/tracks/%1$s";
-    private static final String FAVORITES_NODE = "user/%1$s/";
-    private static final String FAVORITES_BY_ID_NODE = "user/%1$s/favorites/%2$s";
+    private static final String FAVORITES_NODE = "user/%1$s/favorites";
+    private static final String FAVORITES_BY_ID_NODE = "user/%1$s/favorites/map/%2$s";
```

Comment:
`FirebaseUserData` works for me, but then we'll have to map it to its child when you get the data (to maintain the current behavior)